website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -16,64 +16,64 @@ If you are using Appsmith Business Edition, with the Helm chart from `helm.appsm
    appsmithctl backup
    ```
 
-3. Once this is finished, you should have a backup archive generated at a location like this. This is the Appsmith backup archive.
+3. Once the backup process is complete, the backup archive is available at a location like this:
 
    ```
    /appsmith-stacks/data/backup/appsmith-backup-2022-10-24T07-09-56.930Z.tar.gz
    ```
 
-4. Exit out of the shell with:
+4. To exit the shell, run the following command:
    
    ```bash
    exit
    ```
 
-5. Download the backup archive:
+5. To download the backup archive, run the following command:
 
    ```bash
    kubectl cp <namespace>/appsmith-0:<appsmith_backup_archive_path> ./appsmith-ee-backup.tar.gz
    ```
 
-6. Grab the salt and password from the pod, and copy them to `values.yaml`.
+6. To retrieve the salt and password from the pod, run the following command and copy the values to `values.yaml`.
 
    ```bash
    kubectl exec <pod_name> -- grep /appsmith-stacks/configuration/docker.env APPSMITH_ENCRYPTION_
    ```
 
-   Set these values in the `applicationConfig`:
+   Set these values in the `applicationConfig` section:
 
    ```yaml
-     APPSMITH_ENCRYPTION_PASSWORD: "password_here"
-     APPSMITH_ENCRYPTION_SALT: "salt_here"
+     APPSMITH_ENCRYPTION_PASSWORD: <PASSWORD>
+     APPSMITH_ENCRYPTION_SALT: <SALT>
```

Comment:
Can we keep the `"` characters please? Including them is recommended, and with this it's not immediately obvious that they have to wrap the values in double-quotes.